components/schemas/infrastructure/servers/ServerMounts.yml

```diff
@@ -0,0 +1,19 @@
+title: ServerMounts
+type: object
+additionalProperties:
+  type: object
+  required:
+    - type
+    - options
+    - source
+  properties:
+    type:
+      type: string
+      description: String describing the server mount type.
+    options:
+      type: string
+      description: Comma separated string describing the server mount options.
+      example: key1=value1,key2=value2
+    source:
+      type: string
+      description: String describing the IP of the server mount device.
```

Comment:
This isn't correct. This is using the native linux mount under the hood, it doesn't have to be an IP.

![image](https://github.com/cycleplatform/api-spec/assets/5110855/d9cfeb74-38f6-41d9-9811-88b58f93138a)